Code/CIPLabeler/CIPLabeler.h

```diff
@@ -0,0 +1,55 @@
+//
+//
+//  Copyright (C) 2020 Schrödinger, LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#pragma once
+
+#include "boost/dynamic_bitset.hpp"
+
+#include <RDGeneral/export.h>
+
+namespace RDKit {
+
+class ROMol;
+
+namespace CIPLabeler {
+
+/**
+ * Calculate Stereochemical labels based on an accurate implementation
+ * of the CIP rules, described in:
+ *
+ *  Hanson, R. M., Musacchio, S., Mayfield, J. W., Vainio, M. J., Yerin, A.,
```

Comment:
I think it's worth acknowledging here in the function comment that the implementation is derived from centres (by mentioning John M and linking to his github repo)

Reply:
Done.